packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <div
+    :class="leftbarContactCentersRowClasses"
+    data-qa="dt-leftbar-row"
+  >
+    <a
+      class="dt-leftbar-row__primary"
+      :data-qa="$attrs['data-qa'] ?? 'dt-leftbar-row-link'"
+      :aria-label="getAriaLabel"
+      :title="description"
+      :href="$attrs.href ?? 'javascript:void(0)'"
+      v-bind="$attrs"
+      v-on="$listeners"
+      @mouseenter="hovered = true"
+      @focus="hovered = true"
+      @mouseleave="hovered = false"
+      @blur="hovered = false"
+    >
+      <div class="dt-leftbar-row__alpha">
+        <dt-icon
+          name="headphones"
+          size="300"
+          data-qa="dt-leftbar-row-icon"
+        />
```

Comment:
We're migrating to start using single icons instead of `DtIcon` component: https://dialtone.dialpad.com/about/whats-new/posts/2024-4-15.html#using-dialtone-icons

```suggestion
        <dt-icon-headphones
          size="300"
          data-qa="dt-leftbar-row-icon"
        />
```